src/main/resources/edu/cornell/kfs/pmw/cu-ojb-pmw.xml

```diff
@@ -19,6 +19,7 @@
 <descriptor-repository version="1.0">
 
 <class-descriptor class="edu.cornell.kfs.pmw.batch.businessobject.PaymentWorksVendor" table="CU_PMW_VENDOR_T">
+    <field-descriptor name="id" column="ID" jdbc-type="INTEGER" primarykey="true" sequence-name="CU_PMW_VNDR_SEQ" autoincrement="true" />
```

Comment:
I did not see SQL for creating the CU_PMW_VNDR_SEQ, does it already exist?

Reply:
The sequence already exists and was previously created by the archived SQL in [KFSPTS-9246-staging-table.sql]